habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -636,9 +640,13 @@ def step(self, action: Union[str, int]) -> Observations:
             debug_obs = self.get_sensor_observations()
             obs["robot_third_rgb"] = debug_obs["robot_third_rgb"][:, :, :3]
 
+        self.check_add_sim_blob_observation(obs)
+
         if self.habitat_config.HABITAT_SIM_V0.get(
             "ENABLE_GFX_REPLAY_SAVE", False
         ):
+            # sloppy: don't currently support both batch-render and save_keyframe
+            assert not self.habitat_config.BATCH_RENDER
```

Comment:
asserts can be disabled in python for increased speed. Is this a sanity check or should it raise an error? If so, raise a NotImplementedError or something likewise informative.